app/src/main/java/swati4star/createpdf/fragment/ExceltoPdfFragment.java

```diff
@@ -0,0 +1,221 @@
+            try {
+                startActivityForResult(
+                        Intent.createChooser(intent, String.valueOf(R.string.select_file)),
+                        mFileSelectCode);
+            } catch (android.content.ActivityNotFoundException ex) {
+                showSnackbar(mActivity, R.string.install_file_manager);
+            }
+            mButtonClicked = true;
+        }
+    }
+
+    /**
+     * This function opens a dialog to enter the file name of
+     * the converted file.
+     */
+    @OnClick(R.id.create_excel_to_pdf)
+    public void openExcelToPdf() {
+        if (!mPermissionGranted) {
+            getRuntimePermissions();
+            return;
+        }
+        new MaterialDialog.Builder(mActivity)
+                .title(R.string.creating_pdf)
+                .content(R.string.enter_file_name)
+                .input(getString(R.string.example), null, (dialog, input) -> {
+                    if (StringUtils.isEmpty(input)) {
+                        showSnackbar(mActivity, R.string.snackbar_name_not_blank);
+                    } else {
+                        final String inputName = input.toString();
+                        if (!mFileUtils.isFileExist(inputName + getString(R.string.pdf_ext))) {
+                            convertToPdf(inputName);
+                        } else {
+                            MaterialDialog.Builder builder = createOverwriteDialog(mActivity);
+                            builder.onPositive((dialog12, which) -> convertToPdf(inputName))
+                                    .onNegative((dialog1, which) -> openExcelToPdf())
+                                    .show();
+                        }
+                    }
+                })
+                .show();
+    }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        mButtonClicked = false;
+        switch (requestCode) {
+            case mFileSelectCode:
+                if (resultCode == RESULT_OK) {
+                    mExcelFileUri = data.getData();
+                    mRealPath = RealPathUtil.getRealPath(getContext(), data.getData());
+                    showSnackbar(mActivity, getResources().getString(R.string.excel_selected));
+                    String fileName = mFileUtils.getFileName(mExcelFileUri);
+                    if (fileName != null) {
+                        if (fileName.endsWith(Constants.excelExtension))
+                            mFileExtension = Constants.excelExtension;
+                        else if (fileName.endsWith(Constants.excelWorkbookExtension))
+                            mFileExtension = Constants.excelWorkbookExtension;
+                        else {
+                            showSnackbar(mActivity, R.string.extension_not_supported);
+                            return;
+                        }
+                    }
+                    fileName = getResources().getString(R.string.excel_selected)
```

Comment:
I think here it should be `excel_tv_view_text` rather than `R.string.excel_selected`

Reply:
Okay will change that